src/main/java/net/plaaasma/vortexmod/interior/util/InteriorUtil.java

```diff
@@ -0,0 +1,205 @@
+     * Gets the corners of a players interior
+     * TARDIS' will have a size of 10000 in each axis
+     * @param server the current server, used for obtaining data
+     * @param owner the tardis to look for
+     * @return the corners found
+     */
+    public static BlockPos[] getCorners(MinecraftServer server, String owner) {
+        LocationMapData data = LocationMapData.get(server.getLevel(Level.OVERWORLD));
+
+        // This ( at least from what ive read ) should be the greatest coordinate of the interior
+        // Tardis' ( from what ive seen ) will have a size of 10000
+        BlockPos first = data.getDataMap().get(owner);
+
+        if (first == null) {
+            // Bad code.
+            VortexMod.P_LOGGER.warn("Could not find interior for " + owner + " | Giving this TARDIS a new position");
+            data.getDataMap().put(owner, findNewInteriorPosition());
+            first = data.getDataMap().get(owner);
+        }
+
+
+        // We offset the greatest coordinate by the size of the interior, to get the smallest possible coord the interior can have
+        BlockPos second = first.offset(-INTERIOR_SIZE, 0, -INTERIOR_SIZE);
+
+        return new BlockPos[] {first, second};
+    }
+
+    /**
+     * @return The two corners in box form, for better math
+     */
+    public static AABB toBox(BlockPos first, BlockPos second) {
+        return new AABB(first, second);
+    }
+
+    /**
+     * @return A random position from range -10000 to 10000 in the x and z axis
+     */
+    public static BlockPos findNewInteriorPosition() {
+        return new BlockPos(RANDOM.nextInt(-10000, 10000), -128, RANDOM.nextInt(-10000, 10000));
+    }
+
+    /**
+     * Returns whether a position is within a volume of coordinates
+     * @param pos the position to look for
+     * @param first lowest corner of the volume
+     * @param second largest corner of the volume
+     * @return whether the position is within the volume
+     */
+    public static boolean isInArea(BlockPos pos, BlockPos first, BlockPos second) {
+        return pos.getX() >= first.getX() && pos.getX() <= second.getX() && pos.getZ() >= first.getZ() && pos.getZ() <= second.getZ();
+    }
+
+    /**
+     * Returns whether an entity is within a volume of coordinates
+     * @param entity the entity to look for
+     * @param first lowest corner of the volume
+     * @param second largest corner of the volume
+     * @return whether the entity is within the volume
+     */
+    public static boolean isInArea(Entity entity, BlockPos first, BlockPos second) {
+        return isInArea(entity.blockPosition(), first, second);
+    }
+
+    /**
+     * Returns whether a position is within a TARDIS' interior
+     * @param pos the position to look for
+     * @param server server for grabbing location data
+     * @param owner the key in the LocationMapData for a TARDIS ( TODO - I've just realised now that we store interiors using Entity UUIDS, not Owner Hashcodes. Dammit )
```

Comment:
Don't worry about the todo though it should be fine and if its not its a quick fix of replacing all the `String owner` with `UUID tardis`